app/main.py

```diff
@@ -1,3 +1,29 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.information_unpacker import unpacker
+
+
+def shop_trip() -> None:
+    customers, shops, fuel_price = unpacker("app/config.json")
+    for customer in customers:
+        print(f"{customer.name} has {customer.money} dollars")
+        cost_trips = [
+            [
+                shop, round((customer.road_cost(shop.location, fuel_price)
+                             * 2 + shop.cost_cart(customer.product_cart)), 2)
+            ]
+            for shop in shops
+        ]
+        cheap_trip = cost_trips[0]
```

Comment:
What is the need for having this variable here in case `cheap_trip` will be redefined anyway in the for loop without prior use?